tests/user/fixtures.py

```diff
@@ -15,3 +16,7 @@ def user_email():
 @pytest.fixture
 def user_password():
     return 'testpassword'
+
+@pytest.fixture
+def user(user_password, user_email):
+    return get_user_model().objects.create_user(email=user_email, password=user_password)
```

Comment:
`
@pytest.fixture
def create_user(db, django_user_model, test_password):
    def make_user(**kwargs):
        kwargs['password'] = test_password
        if 'email' not in kwargs:
            kwargs['email'] = str(uuid.uuid4()) + '@example.com'
        return django_user_model.objects.create_user(**kwargs)
    return make_user
`